spec/spec_helper.rb

```diff
@@ -2,14 +2,13 @@
 require 'rest-client'
 require 'ten_thousand_feet/client'
 require 'ten_thousand_feet'
-require 'yaml'
 require 'vcr'
 require 'webmock'
+require 'dotenv'
 
-$AUTH = YAML.load_file('auth.yml')
-
+Dotenv.load 'config.env'
```

Comment:
The dotenv README.md says the following:
> env vars that are general to all environments should be stored in .env

Will you please change `config.env` to `.env`? (also pease change the `config.env` line you added to the .gitignore)